modules/__tests__/MemoryHistory-test.js

```diff
@@ -53,17 +53,6 @@ describe('memory history', function () {
 
       expect(location.state).toEqual({ id: 5 })
       expect(location.pathname).toEqual('/5')
-
-      expect(function () {
```

Comment:
Well, if we already have test coverage, can we just leave the cases there (with some comments) just to check that they don't throw?